src/applications/simple-forms/21-10210/sass/10210-lay-witness-statement.scss

```diff
@@ -5,3 +5,34 @@
 @import "~@department-of-veterans-affairs/formation/sass/modules/m-modal";
 @import "~@department-of-veterans-affairs/formation/sass/modules/m-omb-info";
 @import "../../../../platform/forms/sass/m-form-confirmation";
+
+.schemaform-label[for="root_witnessRelationshipToClaimant_family-or-friend"] {
```

Comment:
I assume the scss file, once loaded, could affect every part of vets website?
If so, I think to be safe we might need a form specific prefix in all of our css classes and try to use those classes to target elements? For example the 'div' selector might start looking at every div on vets website if that's true.

Reply:
Yeah, it'd be great to have an ID or classhook for the whole form in the new library!

Here, I've refactored the rules for more specificity.  Also, FYI, this stylesheet only gets imported when the form-app loads.